web/src/styles.module.css

```
@@ -626,6 +626,61 @@
+
+@keyframes spin {
+  to {
+    transform: rotate(360deg);
+  }
+}
+
+.reconnectingSpinner {
+  display: inline-block;
+  width: 14px;
+  height: 14px;
+  border: 2px solid currentColor;
```

Stylelint (17.3.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)